packages/web/src/pages/search-page-v2/SearchPageV2.tsx

```diff
@@ -73,8 +73,8 @@ export const SearchPageV2 = () => {
       setStackReset(true)
 
       const commonFilters = intersection(
-        categories[category].filters,
-        categories[newCategory].filters
+        categories[category]?.filters ?? [],
+        categories[newCategory]?.filters ?? []
```

Comment:
Small fix. Unrelated to PR, but needed to not crash when preserving filters when category changes